sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -10482,367 +9524,1492 @@
+              "calculateValue": "",
+              "calculateServer": false,
+              "allowCalculateOverride": false,
+              "validate": {
+                "required": true,
+                "onlyAvailableItems": true,
+                "customMessage": "Have lived in B.C. for at least 12 consecutive months",
+                "custom": "",
+                "customPrivate": false,
+                "json": "",
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false
+              },
+              "errorLabel": "",
+              "errors": "",
+              "key": "bcResident",
+              "tags": [],
+              "properties": {},
+              "conditional": {
+                "show": "",
+                "when": null,
+                "eq": "",
+                "json": ""
+              },
+              "customConditional": "",
+              "logic": [],
+              "attributes": {
+                "data-cy": "bcResident"
+              },
+              "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+              },
+              "type": "radio",
+              "input": true,
+              "lockKey": true,
+              "placeholder": "",
+              "prefix": "",
+              "suffix": "",
+              "multiple": false,
+              "unique": false,
+              "refreshOn": "",
+              "dataGridLabel": false,
+              "widget": null,
+              "validateOn": "change",
               "showCharCount": false,
               "showWordCount": false,
               "allowMultipleMasks": false,
               "addons": [],
-              "id": "e7vkza"
+              "inputType": "radio",
+              "fieldSet": false,
+              "id": "e3yxtjc",
+              "defaultValue": ""
             },
             {
-              "autofocus": false,
-              "input": true,
-              "tableView": true,
-              "inputType": "radio",
-              "label": "Are you a student with a permanent disability, or a persistent or prolonged disability, and applying for disability grants?",
-              "key": "applicationPDPPDStatus",
-              "values": [
+              "title": "Not a BC resident panel",
+              "breadcrumbClickable": false,
+              "buttonSettings": {
+                "previous": false,
+                "cancel": false,
+                "next": false
+              },
+              "scrollToTop": false,
+              "collapsible": false,
+              "key": "notABcResidentPanel",
+              "conditional": {
+                "show": true,
+                "when": "bcResident",
+                "eq": "no"
+              },
+              "type": "panel",
+              "label": "Not a BC Resident Panel",
```

Comment:
Can you please check the header in this panel?
![image](https://github.com/user-attachments/assets/56233255-0ab5-425f-876a-a204180ebff5)